admin-console/src/main/java/com/agiletec/apsadmin/portal/PageAction.java

```diff
@@ -159,15 +171,14 @@ public String settingsPage() {
     }
 
     protected void valueFormForNew(IPage parentPage) {
-        String groupName = null;
         boolean groupSelectLock = false;
         this.setStrutsAction(ApsAdminSystemConstants.ADD);
         if (parentPage != null) {
             this.setParentPageCode(parentPage.getCode());
-            groupName = parentPage.getGroup();
+            String groupName = parentPage.getGroup();
             this.setGroup(groupName);
             boolean isParentFree = Group.FREE_GROUP_NAME.equals(groupName);
-            groupSelectLock = !(this.isCurrentUserMemberOf(Group.ADMINS_GROUP_NAME) && isParentFree);
+            groupSelectLock = !isParentFree && !this.isCurrentUserMemberOf(Group.ADMINS_GROUP_NAME);
```

Comment:
This is logically different. Just want to make sure that is intentional to fix something?